projects/igniteui-angular/src/lib/core/styles/components/progress/circular/_circular-theme.scss

```diff
@@ -40,24 +48,28 @@
     $theme: digest-schema($circular-bar-schema);
     $meta: map.get($theme, '_meta');
 
-    $progress-circle-color-start: map.get($theme, 'progress-circle-color');
-    $progress-circle-color-end: map.get($theme, 'progress-circle-color');
+    $fill-color-default-start: map.get($theme, 'fill-color-default');
+    $fill-color-default-end: map.get($theme, 'fill-color-default');
```

Comment:
I'm not sure if renaming `progress-circle-color-start` is the best approach, as it now gives the impression that I have variables, such as `fill-color-info-start`, which do not exist.